scripts/ayp.coffee

```diff
@@ -117,8 +117,9 @@ module.exports = (robot) ->
     buffer.store robot.name.trim(), str.trim()
 
   # We listen to everything.
-  # Everything.
-  robot.catchAll (msg) ->
+  # Almost...everything...
+  # Don't listen to robot commands via the alias (.)
+  robot.hear /^(?!.*^\.)/i (msg) ->
```

Comment:
Maybe just because I find look ahead stuff crazy, but I think something like `/^[^\.].*/` is a bit more clear. The start of the string can be anything but `.` and then anything after that.